### ISSUE TITLE
Different version in promp -> prerelease and prerelease

### ISSUE THREAD

**Issue by komalvmehta** [BigstickCarpet/version-bump-prompt#21]
in package.json, version is 4.4.3-0
prerelease version (incorrect)
![image](https://user-images.githubusercontent.com/5324205/30079037-b357e79a-9233-11e7-82f6-18e122654a9f.png)

Shows different on prompt for prerelease version (correct as per semver)
![image](https://user-images.githubusercontent.com/5324205/30079061-cb7e5250-9233-11e7-848c-53fb4b3649ce.png)

**Comment by perry-mitchell** [comment]
I'm seeing the same issue, I think.

```
> bump --prompt --commit --tag --push --all


Current version in package.json is 1.0.0-8
? How would you like to bump it? pre-release (1.0.0-9)
✔ Updated package.json to 1.0.0-8.0
✔ Git commit
```

Running the pre-release option _should_ change the version to `1.0.0-9`, as it mentions, but it instead bumps to `1.0.0-8.0`. No idea where this version came from. I'm using version 4.2.2.

**Comment by JamesMessinger** [comment]
Weird.  That's definitely a bug.  I'll add a test for it in the new version and ensure that it gets fixed

**Comment by JamesMessinger** [comment]
I just released [**version 5.0.0**](https://github.com/JS-DevTools/version-bump-prompt/blob/master/CHANGELOG.md), which resolves this issue.
